### DISCUSSION TITLE
Can i set a name or tip for the port of  the node ?

### DISCUSSION THREAD

**Discussion by YqxLzx** [xyflow/xyflow#2390] [Q&A]
like this ~[
<img width="651" alt="WX20220830-221452@2x" src="https://user-images.githubusercontent.com/68573559/187461096-4702e600-2027-47b7-9ae2-be51f7c0b022.png">
](url)
thanks ~

**Comment by moklick** [comment]
Yep. The nodes are divs you can pass child nodes and position them with css.
